src/Routers/AppRouter.tsx

```diff
@@ -75,7 +90,14 @@ export default function AppRouter() {
   }
 
   useRedirect("/user", "/users");
-  const pages = useRoutes(routes) || <Error404 />;
+
+  const allRoutes = {
+    ...routes,
+    ...pluginRoutes,
```

Comment:
Can't we just merge it with `routes`, ie add `...pluginRoutes` to line`:58`

Reply:
As of now, pluginRoutes are state based, so cannot be kept along with the reset. We'll need dynamically flippable plugins at some point, given that I would leave it as is. 

We can rename `routes => baseRoutes` and `allRoutes => routes`